### PR TITLE
Refactor home page

### PR DESCRIPTION
**Pull request by arun1595**

### CODE REVIEW

**Comment by KevinMulhern** [comment]
LGTM :shipit:

File: app/views/shared/_navbar.html.erb [hunk 1]
```diff
@@ -1,37 +1,43 @@
 <div class="navbar navbar-toggleable-md navbar-light">
-  <button class="navbar-toggler navbar-toggler-right" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+  <button class="navbar-toggler navbar-toggler-right" type="button" data-toggle="collapse" data-target="#navbar-supported-content" aria-controls="navbar-supported-content" aria-expanded="false" aria-label="toggle-navigation">
```
Comment (KevinMulhern) [inline app/views/shared/_navbar.html.erb]: With these being bootstrap classes, will this break the navbar by using different names?

Comment (arun1595) [inline app/views/shared/_navbar.html.erb]: No.

**Comment by arun1595** [comment]
@KevinMulhern Addressed your comments. PTAL :)

**Comment by KevinMulhern** [comment]
LGTM 👍 :shipit:

**Pull request merged**